editor/editor_help.cpp

```diff
@@ -2862,9 +2862,17 @@ void EditorHelpTooltip::parse_tooltip(const String &p_text) {
 	const String &property_name = slices[2];
 	const String &property_args = slices[3];
 
+	String formatted_text;
+
+	// Exclude internal properties, they are not documented.
+	if (type == "internal_property") {
+		formatted_text = "[i]" + TTR("This property can only be set in the Inspector.") + "[/i]";
```

Comment:
I do think "This property can only be set in the Inspector" is confusing. To me, it just suggests that the user shouldn't change it with a script, but they should feel free to change it in the inspector. If that's what is intended, fine, but I thought the intention was that the user should not touch this property at all. If that's the intention, maybe something like "This is an internal property. It should not be modified by the user."

Reply:
> To me, it just suggests that the user shouldn't change it with a script, but they should feel free to change it in the inspector.

That's exactly the intention. Why would we want to have a property visible and editable, but tell people to not touch it?

Reply:
I thought maybe it was a quirk of the engine that certain properties just needed to be exposed for stuff to work correctly 🤷 

If that's the case, the tooltip should provide some description of what the property actually does no? It seems like either the user is not supposed to interact with the property, so no description is required OR the user is encouraged to edit the property from the inspector, in which case a description seems absolutely necessary.

Reply:
> If that's the case, the tooltip should provide some description of what the property actually does no?

Yes, ideally, but as discussed above in the comments, we currently don't have any mechanism to document internal properties because by design they are not supposed to be documented.

Reply:
I see. Are all internal properties like `anchors_preset` in that there is a `set_anchors_preset` method that the user is supposed to use instead of setting the property itself? If so, maybe there's a way to inject the documentation for the appropriate setter method into the tooltip, or at least a message like "see `set_anchors_preset` method for details."

In any case, if that's beyond the scope of this PR, what you're doing now looks good to me and the code seems fine. As you can tell from my comments, I don't know a lot about this TIWAGOS :)

Reply:
`set_anchors_preset` is unrelated to the `anchors_preset` property, they just have a name overlap. Functionally they are related, of course, since both deal with presets for anchors.

That said, the potential description for `anchors_preset` would include references to all existing `set_*_preset` methods.